MIP41 Amendments to smooth out interim facilitator processes. (#711)
makerdao/mips: MIP41/mip41.md

---

### MIP41c6: Interim Facilitators

#### Interim Facilitators

Whenever a Core Unit finds itself with no Facilitator, an Interim Facilitator is needed to ensure the continuity or graceful wind-down of the Core Unit.

> A Retiring Faciliator may nominate a single Interim Facilitator. Subject to the nominated Interim Facilitator's acceptance, they are immediately ratified as the Interim Facilitator.

Reviewer: What's the rationale for restricting to a single interim facilitator? It's obviously easy to administer, but it decreases optionality.

> The Facilitators of other Core Units may elect an alternative Interim Facilitator within 72 hours of the initial Facilitator Offboarding using the Interim Facilitator Election Process. In the event an alternative Interim Facilitator is elected, they immediately replace the current Interim Facilitator, if one exists.

Reviewer: Is this going to be the standard process if there is either no interim nominated by the retiring facilitator, or the previous facilitator has been offboarded by Governance without a new interim being proposed? If so, I'd state it explicitly

Reviewer: I should really read the whole thing before I start commenting

The incentive to volunteer as an Interim Facilitator is the compensation tied to a Core Unit unwinding plan. In the event of a missing, failed succession, or headless core unit, the incentive to volunteer as an Interim Facilitator is a transfer of the former Facilitator's compensation during the interim period.

#### Interim Facilitator Responsibilities And Powers

> An Interim Facilitator is empowered to serve in this role for a maximum of two months. 

Reviewer: I think MIP41c4 should override an interim facilitator, probably best to specify this.

```suggestion
An Interim Facilitator is empowered to serve in this role for a maximum of two months. If a new permanent Facilitator is onboarded to the relevant Core Unit through a MIP41c4 subproposal they will be immediately off-boarded.
```

In the event of a Core Unit offboarding, the Interim Facilitator is responsible for overseeing and executing the unwinding process. 

In the event of a Facilitator offboarding, the Interim Facilitator is responsible for both:
* Ensuring the functional continuity of the Core Unit during the interim period.

> * Making a good faith effort to deliver one or more viable Facilitator Onboarding subproposals for the Core Unit to Maker Governance.

Reviewer: Might be unnecessary but might be worth clarifying

```suggestion
* Making a good faith effort to deliver one or more viable Facilitator Onboarding subproposals for the Core Unit to Maker Governance. The Interim Facilitator is welcome to propose themselves as permanent Facilitator if they wish.
```

The Interim Facilitator is empowered to use the Facilitator Governance Powers defined in MIP41c2 to keep existing operational processes running smoothly during the interim period.

#### Interim Facilitator Election Process

The Interim Facilitator Election Process assumes the existence of one or more permanent Governance Facilitators. If this is not the case, MIP41c6 must be considered non-binding.

> An Interim Facilitator may be elected by a majority of Core Units voting in favour of a given individual. The individual must be willing to take the role to be eligible for election. Each Core Unit may cast a single vote for or against each candidate. In the event of a Core Unit containing multiple facilitators, they must come to consensus on their Core Unit's votes before communicating them publicly.

Reviewer: ```suggestion
An Interim Facilitator may be elected by a majority of Core Units voting in favour of a given individual. The individual must be willing to take the role to be eligible for election. A majority of Governance Facilitators should decide and publicly declare the nomination process. For operational reasons, this process may be carried out differently depending on the circumstances. Each Core Unit may cast a single vote for or against each candidate. In the event of a Core Unit containing multiple facilitators, they must come to consensus on their Core Unit's votes before communicating them publicly.
```

> In the absence of succesful Interim Facilitator selection or election within 72 hours, one or more permanent facilitator onboarding proposals may be made under the next weekly governance cycle. This overrides the normal rules relating to the feedback and frozen periods.

Reviewer: "next weekly governance cycle" may be impractical if this all kicks off on a weekend for example, make it slightly more vague to allow a delay until the next cycle if necessary

```suggestion
In the absence of succesful Interim Facilitator selection or election within 72 hours, one or more permanent facilitator onboarding proposals may be expedited through the weekly governance cycle. This overrides the normal rules relating to the feedback and frozen periods.
```

The Governance Facilitators are responsible for the implementation of this proces. A majority of  Governance Facilitators may modify this process to suit the circumstances by communicating details and reasoning publicly prior to the beginning of the process. 

Core Unit Facilitators should refrain from casting votes prior to the Governance Facilitators publicly communicating the details of the process's implementation. 

> 
> 

Reviewer: Remove extra line break

```suggestion

```

#### Edge Cases

If a Retiring Facilitator has not nominated an Interim Facilitator, the other Core Unit Facilitators must use the Interim Facilitator Election Process to elect an Interim Facilitator.

> If a permenant facilitator is missing and uncontactable without notice and for unknown reasons, a majority of Governance Facilitators must use their best judgement as how to proceed with this process.

Reviewer: spelling

```suggestion
If a permanent facilitator is missing and uncontactable without notice and for unknown reasons, a majority of Governance Facilitators must use their best judgement as how to proceed with this process.
```

If a Core Unit has no active Facilitators as a result of involuntary offboarding and has no active budget, no Interim Facilitator will be appointed unless there are winding down activities that require one.

> If a permament Facilitator has not been chosen prior to the expiry of an Interim Facilitator's two month term, the Core Unit is offboarded, and any budgets revoked. A majority of Governance Facilitators may extend this time-limit by stating publicly that they are doing so, and the reason they are doing so. 

Reviewer: spelling and slight syntax change

```suggestion
If a permanent Facilitator has not been chosen prior to the expiry of an Interim Facilitator's two-month term, the Core Unit is offboarded, and any budgets revoked. A majority of Governance Facilitators may extend this time-limit by stating publicly that they are doing so, and the reasoning behind this decision. 
```

> If the full Interim Facilitator Election Process (including the expedited weekly cycle permenent facilitator subproposals) fails to install either an Interim or permenant Facilitator, the Core Unit should be considered offboarded, and any budgets revoked. Appropriate MIP subproposals should be filled out by the Governance Facilitators and marked as Approved without going to vote.

Reviewer: sp

```suggestion
If the full Interim Facilitator Election Process (including the expedited weekly cycle permanent facilitator subproposals) fails to install either an Interim or permanent Facilitator, the Core Unit should be considered offboarded, and any budgets revoked. Appropriate MIP subproposals should be filled out by the Governance Facilitators and marked as Approved without going to vote.
```

In the event of a Facilitator offboarding, the Interim Facilitator is responsible for both:
* Ensuring the functional continuity of the Core Unit during the interim period.
* Making a good faith effort to deliver one or more viable Facilitator Onboarding subproposals for the Core Unit to Maker Governance. The Interim Facilitator may propose themselves as a permanent Facilitator.

The Interim Facilitator is empowered to use the Facilitator Governance Powers defined in MIP41c2 to keep existing operational processes running smoothly during the interim period.

If the Core Unit is not being off-boarded, an Interim Facilitator receives DAI compensation monthly equal to the salary of the previous permanent Facilitator if this information is publicly available, otherwise 15,000 DAI per month. 

When a Core Unit is offboarded if an Interim Facilitator is judged to be necessary to wind down the Core Unit gracefully, then compensation for an Interim Facilitator must be included as part of the offboarding subproposal.

---

### MIP41c7: Interim Facilitator Appointment Process 

> The Interim Facilitator Appointment Process assumes the existence of one or more permanent Governance Facilitators. If this is not the case, MIP41c6 must be considered non-binding. The Governance Facilitators are responsible for the implementation of this process. A majority of Governance Facilitators may modify this process to suit the circumstances by communicating details and reasoning publicly before the process begins.

Reviewer: This should be MIP41c7 now that we've split out the component, right?

```suggestion
The Interim Facilitator Appointment Process assumes the existence of one or more permanent Governance Facilitators. If this is not the case, MIP41c7 must be considered non-binding. The Governance Facilitators are responsible for the implementation of this process. A majority of Governance Facilitators may modify this process to suit the circumstances by communicating details and reasoning publicly before the process begins.
```
